src/exampleTypes/TagsQuerySearchBar.js

```diff
@@ -82,7 +89,11 @@ let SearchBar = ({ tree, node, resultsPath, actionWrapper }) => {
           />
         </Box>
         {tree.disableAutoUpdate && (
-          <SearchButton tree={tree} resultsPath={resultsPath} />
+          <SearchButton
```

Comment:
This is a very specific use case. Can't we parametrize `SearchButton` itself?

Reply:
@stellarhoof can you elaborate more how that is better and different that this? Also what is the current issue you see if we pass another property to the `TagsQuerySearchBar`. Not sure I follow how the very specific use case is a problem here. It is just another prop passed through to eventually the `SearchButton`.

Reply:
Here's how I usually look at this: you've got a component with internal components and you want to forward some props to the internal ones. The line between doing that or parametrizing the component wholesale is kinda blurry. Keeping everything else constant, if you look at the component API and the prop feels kinda arbitrary then you might be better off passing the component itself.

If I see `<TagsQuerySearchBar buttonAttributes={buttonAttributes}/>` my first reaction would be 1) Oh that's kinda specific and 2) Which button gets this? 3) Why not just button props? etc... whereas this `<TagsQuerySearchBar SearchButton={<SearchButton ... />} />` is self-evident. Of course, the second usage would be annoying if we had to use it like that everywhere, but given that this is a very specific use case I think the second approach is better.

Reply:
@stellarhoof we would have to use it everywhere :). In the given request they want to add attributes to the search button for all pages. So spending/bids/contacts would have to have the SearchButton passed ... and we practically do not use anywhere else the `TagsQuerySearchBar`. I agree with your reasoning just not sure if in this case it makes sense since instead of passing an attribute I have to now pass the entire button. Not to mention that the actual `SearchButton` is not even exported from `contexture-react` and we would have to do that as well.

Reply:
True, and I'd have no feedback if this was on the bid-search side, but contexture-react is a bit more generic, so having a prop like this for a specific use case on our main product is kinda yucky

Reply:
OK @michsa do you agree as well on @stellarhoof suggestion? If you guys both are on the same opinion I would obviously listen and change it :)

Reply:
Changes made as per the discussion